src/engraving/types/types.h

```diff
@@ -1087,6 +1087,24 @@ enum class TieDotsPlacement {
     AFTER_DOTS
 };
 
+enum class TimeSigPlacement {
```

Comment:
Let's make these enums ` : char` (as in `enum class TimeSigPlacement : char {`)
The advantage is less memory usage: three possible values also fit in one byte rather than four.